### PR TITLE
Ghost support

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

**Comment by cquiroz** [comment]
Actually this should be a Draft PR

**Comment by cquiroz** [comment]
This is ready to review

File: modules/server/src/test/scala/seqexec/server/tcs/TcsControllerEpicsCommonSpec.scala
Comment (jluhrs) [inline modules/server/src/test/scala/seqexec/server/tcs/TcsControllerEpicsCommonSpec.scala:1]: No test for defocus?

**Review by jluhrs — APPROVED** [review]
LGTM

File: modules/server/src/test/scala/seqexec/server/tcs/TcsControllerEpicsCommonSpec.scala
Comment (cquiroz) [reply modules/server/src/test/scala/seqexec/server/tcs/TcsControllerEpicsCommonSpec.scala:1]: Good point. I think we should as a minimum to have regression testing
I'll plan to add a test in another PR

**Pull request merged**